app/models/course_user.rb

```diff
@@ -41,6 +41,11 @@ class CourseUser < ActiveRecord::Base
     include CourseUser::AchievementsConcern
   end
   has_one :invitation, class_name: Course::UserInvitation.name, dependent: :destroy, validate: true
+  has_many :course_group_users, class_name: Course::GroupUser.name,
```

Comment:
`has_many :group_users`, remove the course namespace so it's consistent with other associations.